mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -738,6 +738,18 @@ func.func @extract_element_with_value_1d(%arg0: vector<16xf32>, %arg1: index) ->
 
 // -----
 
+func.func @extract_element_with_value_2d(%arg0: vector<1x16xf32>, %arg1: index) -> f32 {
+  %0 = vector.extract %arg0[0, %arg1]: f32 from vector<1x16xf32>
+  return %0 : f32
+}
+
+// Multi-dim vectors are not supported but this test shouldn't crash.
```

Comment:
[nit] Could we remove "but this test shouldn't crash."? Pretty much every test makes sure that compiler doesn't crash for some specific case 😅  Instead, we should document what unique code path is being exercised - that's already documented via function name.

I know I am a bit picky, but having reviewed quite a few tests in the context of scalable vectors, I've noticed a lot of duplication and tests documented as "make sure the compiler doesn't crash". Such tests make it very hard to understand what unique path is being exercised (and that's probably why we have so much duplication). Btw, your tests are always super clear :)

Reply:
I find value in "this shouldn't crash" comments. This marks the test as "regression" and differentiates it from the case when something used to be an error with `expected-error` test before support was added removing the error check but not the test. If repeated "this shouldn't crash" comment are too frequent, we may consider having "dialect-foo-regression.mlir` test file that collects them.

Reply:
I agree with the regression point. The comment is helpful for the future reader to answer the question of "what is the goal of this test given that the vector.extract is not lowered to LLVM?". I haven't seen this kind of tests added redundantly as they are added as a result of a crash (so if someone hits a crash it's probably not covered by the existing tests).

We also have plenty of these tests all over the place so I wouldn't like to be a one-off just for this specific case, mostly because the absence of guidelines for regression tests would lead to people missing the "-unsupported" file and adding regression tests to this file instead. Perhaps we should define some guidelines in that regard...

In any case, let me know if you feel super strong about this and I'll create the new file. I'm just trying to be consistent with what we have today.